viewer/packages/camera-manager/src/Flexible/FlexibleControls.ts

```diff
@@ -141,6 +155,14 @@ export class FlexibleControls extends EventDispatcher<FlexibleControlsEvent> {
     return this.options.controlsType;
   }
 
+  set getPickedPointByPixelCoordinates(getPickedPointByPixelCoordinates: GetPickedPointByPixelCoordinates | undefined) {
```

Comment:
We should not allow to set an `undefined` value in setter function

Reply:
Fixed

Reply:
Not fixed. I couldn't do it because of the compiler error